ui/plugins/perf_details/main.html

```diff
@@ -1,15 +1,4 @@
-<div class="talos-panel">
-  <ul ng-repeat="line in job_details"
-      ng-if="line.content_type == 'TalosResult'">
-
-    <li>Graphserver:
-      <ul>
-        <li ng-repeat="gr in line.value.graphserver | toArray | orderBy: '$key'">
-          {{::gr.$key}}: <a href="{{::gr.url}}" target="_blank">{{::gr.result}}</a>
```

Comment:
I am happy to see graphserver links being removed!  Is there other code we can remove from treeherder related to line.value.graphserver?  I would prefer a new bug to address that.